src/components/modules/paste.ts

```diff
@@ -543,10 +585,16 @@ export default class Paste extends Module {
             break;
         }
 
-        const { tags } = tool.pasteConfig;
+        const { tags: tagsOrSanitizeConfigs } = tool.pasteConfig;
+
+        const toolTags = tagsOrSanitizeConfigs.reduce((result, tagOrSanitizeConfig) => {
+          const tags = this.getTags(tagOrSanitizeConfig);
+
+          tags.forEach((tag) => {
+            const sanitizationConfig = _.isObject(tagOrSanitizeConfig) ? tagOrSanitizeConfig[tag] : null;
```

Comment:
This variable will always return `undefined` or `null` because `tag` is uppercased (`IMG`), but keys of `tagOrSanitizeConfig` (`img`) are not

<img width="408" alt="image" src="https://user-images.githubusercontent.com/3684889/197407172-acb1b198-7e32-44bc-926b-2c2be9a57573.png">